android/src/main/java/com/thebluealliance/androidclient/renderers/MatchRenderer.java

```diff
@@ -120,11 +123,53 @@ public MatchRenderer(APICache datafeed, Resources resources) {
 
         long matchTime = match.getTime() != null ? match.getTime() : -1;
 
+        int redExtraRp = 0;
+        int blueExtraRp = 0;
+
+        if(match.getYear() >= 2016) {
+            JsonObject scoreBreakdown = JSONHelper.getasJsonObject(match.getScoreBreakdown());
+            JsonObject redScoreBreakdown = scoreBreakdown.get("red").getAsJsonObject();
```

Comment:
I believe this can throw if the `scoreBreakdown` object doesn't have a key named `red`